src/controllers/issuer.ts

```diff
@@ -64,6 +98,46 @@ export class IssuerController {
     }
   }
 
+  /**
+   * @openapi
+   * 
+   * /key/{kid}:
+   *   get:
+   *     tags: [ Key ]
+   *     summary: Fetch a key pair.
+   *     security: [ bearerAuth: [] ]
+   *     parameters:
+   *       - name: kid
+   *         in: path
+   *         schema:
+   *           type: string
+   *         required: true
+   *     responses:
+   *       200:
+   *         description: The request was successful.
+   *         content:
+   *           application/json:
+   *             schema: 
+   *               $ref: '#/components/schemas/KeyResult'
+   *       400:
+   *         description: A problem with the input fields has occurred. Additional state information plus metadata may be available in the response body.
```

Comment:
Better to create a $ref for status code's 400 and 500, similar to 401. So that we can reuse it